docs/concepts/glossary.zh.md

```diff
@@ -25,142 +25,92 @@ under the License.
-or [partitions](#partition) of data streams or data sets.
+Physical graph 是一个在分布式运行时，把 [Logical Graph](#logical-graph) 转换为可执行的结果。节点是 [Tasks](#task)，边表示数据流或数据集的输入/输出关系或 [partitions](#partition)。
 
 #### Record
 
-Records are the constituent elements of a data set or data stream. [Operators](#operator) and
-[Functions](#Function) receive records as input and emit records as output.
+Records 是数据集或数据流的组成元素。[Operators](#operator) 和 [Functions](#Function)接收 records 作为输入，并将 records 作为输出发出。
 
 #### Flink Session Cluster
 
-A long-running [Flink Cluster](#flink-cluster) which accepts multiple [Flink Jobs](#flink-job) for
-execution. The lifetime of this Flink Cluster is not bound to the lifetime of any Flink Job.
-Formerly, a Flink Session Cluster was also known as a Flink Cluster in *session mode*. Compare to
-[Flink Application Cluster](#flink-application-cluster).
+长时间运行的 [Flink Cluster](#flink-cluster)，它可以接受多个 [Flink Jobs](#flink-job) 的执行。此 [Flink Cluster](#flink-cluster) 的生命周期不受任何 [Flink Jobs](#flink-job) 生命周期的约束限制。以前，Flink Session Cluster 在 *session mode* 中也称为 [Flink Cluster](#flink-cluster)，和 [Flink Application Cluster](#flink-application-cluster) 作对比。
 
 #### State Backend
 
-For stream processing programs, the State Backend of a [Flink Job](#flink-job) determines how its
-[state](#managed-state) is stored on each TaskManager (Java Heap of TaskManager or (embedded)
-RocksDB) as well as where it is written upon a checkpoint (Java Heap of
-[Flink Master](#flink-master) or Filesystem).
+对于流处理程序，[Flink Job](#flink-job) 的 State Backend 决定了其 [state](#managed-state) 是如何存储在每个 TaskManager（ TaskManager 或（嵌入式）RocksDB 的 Java 堆）上，以及它在 checkpoint 上的写入位置（ [Flink Master](#flink-master) 或 Filesystem 的 Java 堆）。
 
 #### Sub-Task
 
-A Sub-Task is a [Task](#task) responsible for processing a [partition](#partition) of
-the data stream. The term "Sub-Task" emphasizes that there are multiple parallel Tasks for the same
-[Operator](#operator) or [Operator Chain](#operator-chain).
+Sub-Task 是负责处理数据流 [Partition](#partition) 的 [Task](#task)。"Sub-Task"是一个强调相同的 [Operator](#operator) 或者 [Operator Chain](#operator-chain) 具有多个并行 Tasks 的术语。
 
 #### Task
 
-Node of a [Physical Graph](#physical-graph). A task is the basic unit of work, which is executed by
-Flink's runtime. Tasks encapsulate exactly one parallel instance of an
-[Operator](#operator) or [Operator Chain](#operator-chain).
+Task 是 [Physical Graph](#physical-graph) 的节点。它是基本的工作单元，由 Flink 的 runtime 来执行。Tasks 正好封装了一个 [Operator](#operator) 或者 [Operator Chain](#operator-chain) 的 *parallel instance*。 
 
 #### Flink TaskManager
 
-TaskManagers are the worker processes of a [Flink Cluster](#flink-cluster). [Tasks](#task) are
-scheduled to TaskManagers for execution. They communicate with each other to exchange data between
-subsequent Tasks.
+TaskManagers 是 [Flink Cluster](#flink-cluster) 的工作进程。[Tasks](#task) 被调度到 TaskManagers 上执行。TaskManagers 相互通信，只为在后续的 Tasks 之间交换数据。
```

Comment:
It should not be plural when translated.

Reply:
```
TaskManager 是 [Flink Cluster](#flink-cluster) 的工作进程。[Tasks](#task) 被调度到 TaskManager 上执行。TaskManager 之间相互通信，以在后续的 Task 之间交换数据。
```

Reply:
Thanks for your reminding, you are right, I noticed the following instructions: 
![image](https://user-images.githubusercontent.com/18230501/62992110-ca452380-be84-11e9-8576-4707a07a96bd.png)